app/models/manageiq/providers/redhat/infra_manager/vm/operations/power.rb

```diff
@@ -4,7 +4,11 @@ def validate_pause
   end
 
   def raw_start
-    with_provider_object(&:start)
+    start_with_cloud_init = custom_attributes.find_by(:name => "miq_provision_boot_with_cloud_init")
+    with_provider_object do |rhevm_vm|
+      rhevm_vm.start { |action| action.use_cloud_init(true) if start_with_cloud_init }
+    end
```

Comment:
No, that puts it in the native provider itself, which it sort of already has via the start method.  I guess I'm wondering why this code goes directly to the native provider instead of calling `.start` or `.raw_start` on the model.  If it's on the model you can add a cloud_init option or a separate method there.